sklearn/feature_extraction/image.py

```diff
@@ -338,6 +338,18 @@ def _extract_patches(arr, patch_shape=8, extraction_step=1):
     return patches
 
 
+@validate_params(
+    {
+        "image": ["array-like"],
+        "patch_size": [tuple],
```

Comment:
This is another case where `tuple` and `list` works, but the parameter validation restricts it to `tuple`. I'm okay with letting this be any `Sequence` of ints with length 2.

@glemaitre @jeremiedbb  What do you think?

Reply:
I wouldn't name it ``Sequence`` since a string is a sequence for instance. I'm okay with ``[tuple, list]``, without forgetting to modify the docstring.

Reply:
+1 for `[tuple, list]`. It's simple and explicit.